typescript/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@process-engine/consumer_api_client",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Minor, nicht patch 😉 

```suggestion
  "version": "6.1.0",
```

---

🤦‍♂